File: Modul-4/Soal-5/PRAK405-2410817320001-NazlaSalsabila.c
#include <stdio.h>

void hitung_kelipatan() {
    int n, kelipatan, total_keseluruhan = 0;

    printf("Masukkan banyaknya n: ");
    scanf("%d", &n);
    printf("Masukkan kelipatan: ");
    scanf("%d", &kelipatan);

    for (int i = 1; i <= n; i++) {
        int total_baris = 0;
        printf("(");
        for (int j = i; j > 0; j--) {
            int hasil = j * kelipatan;
            total_baris += hasil;
            printf("%d * %d", j, kelipatan);
            if (j > 1) {
                printf(") + (");
            }
        }
        printf(") = %d\n", total_baris);
        total_keseluruhan += total_baris;
    }
    printf("%d\n", total_keseluruhan);
}

int main() {
    hitung_kelipatan();
    return 0;
}

File: Modul-4/Soal-5/PRAK405-2410817320001-NazlaSalsabila.py
def hitung_kelipatan():
    n = int(input("Masukkan banyaknya n: "))
    kelipatan = int(input("Masukkan kelipatan: "))
    total_keseluruhan = 0
    for i in range(1, n + 1):
        total_baris = 0
        ekspresi = ""
        for j in range(i, 0, -1):
            hasil = j * kelipatan
            total_baris += hasil
            ekspresi += f"({j} * {kelipatan})"
            if j > 1:
                ekspresi += " + "
        print(f"{ekspresi} = {total_baris}")
        total_keseluruhan += total_baris
    print(total_keseluruhan)

hitung_kelipatan()